recognition/ISICs_Unet/data.py

```diff
@@ -0,0 +1,127 @@
+import numpy as np
```

Comment:
Code header block should be added for all scripts/source files

Reply:
Code header blocks are added in the new pull request to correct branch